build.cake

```diff
@@ -98,60 +97,53 @@ Task("Externals-Android")
+    var macosFrameworksLocation = System.IO.Path.Combine(AppleExternals, "AppCenter-SDK-Apple/macOS");
+
+    // Move iOS frameworks.
+    var iosExternals = System.IO.Path.Combine(AppleExternals, "ios");
+    CleanDirectory(iosExternals);
+
+    // Copy the AppCenter binaries directly from the frameworks and add the ".a" extension.s
+    var files = GetFiles($"{iosFrameworksLocation}/*.framework/AppCenter*");
     foreach (var file in files)
     {
         var filename = file.GetFilename();
-        MoveFile(file, $"{IosExternals}/{filename}.a");
+        MoveFile(file, $"{iosExternals}/{filename}.a");
     }
     
     // Copy Distribute resource bundle and copy it to the externals directory.
     var distributeBundle = "AppCenterDistributeResources.bundle";
-    if(DirectoryExists($"{frameworksLocation}/{distributeBundle}"))
+    if(DirectoryExists($"{iosFrameworksLocation}/{distributeBundle}"))
     {
-        MoveDirectory($"{frameworksLocation}/{distributeBundle}", $"{IosExternals}/{distributeBundle}");
+        MoveDirectory($"{iosFrameworksLocation}/{distributeBundle}", $"{iosExternals}/{distributeBundle}");
     }
-}).OnError(HandleError);
-
-// Downloading macOS binaries.
-Task("Externals-MacOS")
-    .Does(() =>
-{
-    CleanDirectory(MacosExternals);
-    CopyDirectory($"{IosExternals}/AppCenter-SDK-Apple", $"{MacosExternals}/AppCenter-SDK-Apple");
 
-    var frameworksLocation = System.IO.Path.Combine(MacosExternals, "AppCenter-SDK-Apple/macOS");
-
-    // Copy the AppCenter binaries directly from the frameworks and add the ".a" extension
-    var files = GetFiles($"{frameworksLocation}/*.framework/Versions/A/AppCenter*");
-    foreach (var file in files)
+    // Move macOS frameworks.
+    var macosExternals = System.IO.Path.Combine(AppleExternals, "macos");
+    CleanDirectory(macosExternals);
+    var frameworks = GetDirectories($"{macosFrameworksLocation}/*.framework");
+    foreach (var frameworkDir in frameworks)
     {
-        var filename = file.GetFilename();
-        CopyFile(file, $"{MacosExternals}/{filename}.a");
+        var dirName = frameworkDir.GetDirectoryName();
+        MoveDirectory(frameworkDir, $"{macosExternals}/{dirName}");
     }
-
-    //generate correct .framework directories
-    CopyDirectory($"{frameworksLocation}/AppCenter.framework/Versions/A", $"{MacosExternals}/AppCenter.framework");
-    CopyDirectory($"{frameworksLocation}/AppCenterAnalytics.framework/Versions/A", $"{MacosExternals}/AppCenterAnalytics.framework");
-    CopyDirectory($"{frameworksLocation}/AppCenterCrashes.framework/Versions/A", $"{MacosExternals}/AppCenterCrashes.framework");
 }).OnError(HandleError);
 
-
-
 // Create a common externals task depending on platform specific ones
-Task("Externals").IsDependentOn("Externals-Ios").IsDependentOn("Externals-MacOS").IsDependentOn("Externals-Android");
+Task("Externals").IsDependentOn("Externals-Apple").IsDependentOn("Externals-Android");
```

Comment:
Please update task name in the` appcenter-post-clone` scripts and in the `...Bindings.csproj` files for Mac and iOS.